code/datums/observation/logged_out.dm

```diff
@@ -0,0 +1,22 @@
+//	Observer Pattern Implementation: Logged Out
+//		Registration type: /mob
+//
+//		Raised when: A mob logs out (client either logged out or was moved to another mob)
+//
+//		Arguments that the called proc should expect:
+//			/mob/leaver:    The mob that has logged out
+//			/client/client: The mob's client
+
+GLOBAL_DATUM_INIT(logged_out_event, /singleton/observ/logged_out, new)
```

Comment:
No observables, use signals

Reply:
Forgot that I had this in from the click handler PR, will remove shortly